CLAUDE.md

```
@@ -52,16 +52,46 @@ Template variables use Handlebars syntax `{{variableName}}` and are expanded at
 1. `--var key=value` flags (highest priority, repeatable)
 2. `--var-file path` contents (YAML format)
 3. `.rundown/config.yaml` (auto-discovered from cwd upward, stops at git root)
-
-**Example:**
+4. Frontmatter `vars:` field
+5. Built-in defaults (lowest priority)
+
+**Built-in Variables:**
+| Variable | Example Value | Description |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)